File: Flying Island/Assets/flying_island/skybox/changeSkybox.js
#pragma strict

 
//public var space_skybox : Material; 
//public var Timer : float = 10f;


//space_skybox = Resources.Load("space_skybox.mat", typeof(Material)) as Material;

function OnTriggerEnter(other : Collider){
    RenderSettings.skybox = Resources.Load("space_skybox") as Material;
    Debug.Log ('TESTR0');
}

File: Flying Island/Assets/flying_island/skybox/changeSkybox.cs
using UnityEngine;
using System.Collections;

public class changeSkybox : MonoBehaviour {

    public Material night_skybox;
    public Material space_skybox;    

    // Use this for initialization
    void Start () {
        RenderSettings.skybox = night_skybox;
    }
	
	// Update is called once per frame
	void Update () {
	
	}

    void OnTriggerStay(Collider Other)
    {
        RenderSettings.skybox = space_skybox;

        Debug.Log("Changer Skybox");
    }

    void OnTriggerExit(Collider Other)
    {
        Debug.Log("Out");
    }
}
